### PR TITLE
More work on taxonomy writeup

### PR DESCRIPTION
**Pull request by jar398**

### CODE REVIEW

File: doc/method/sources.md [hunk 1]
```diff
@@ -1,77 +1,169 @@
...
+The ranking can be a column in the table (although if the table is in
+rank order, it would just be sequential numbers).
 
-*SILVA*  
-The classification of prokaryotes and unicellular Eukaryotes in NCBI and GBIF is not consistent with current phylogenetic thinking. We therefore imported the SILVA taxonomy, which is built algorithmically based on clustering of molecular
-sequences.  We incorporated SILVA version 115 into OTT 2.10.  We did
-not include SILVA's plant, animal, or fungi branches in OTT
-[references]. SILVA has higher priority than NCBI or GBIF in order to capture the deep relationships in the tree.
+The following information might also go in the table: total number of
+taxa, number of exclusive taxa.
```
Comment (kcranston) [inline doc/method/sources.md]: Some good suggestions here for things to include. Too much for one table, so one way to prune is to include only basic stats here, not results (i.e. number of taxa = yes; exclusive taxa = no).

File: doc/method/sources.md [hunk 2]
```diff
@@ -1,77 +1,169 @@
...
+
+The particular version of NCBI taxonomy used in OTT 2.10 was
+downloaded from NCBI on [date].  [maybe put dates in table??]
+
+
+*GBIF backbone taxonomy*  
+The GBIF backbone taxonomy provides taxonomic coverage - 1.6 million
+species (binomials).  The GBIF backbone draws from a number of sources
+(including IRMNG and Index Fungorum), has ongoing institutional
+support for maintenance and growth, and provides access without
```
Comment (kcranston) [inline doc/method/sources.md]: do we also need to say something about institutional support for NCBI?

Comment (jar398) [inline doc/method/sources.md]: (oops I meant ITIS, not IPNI)

Comment (jar398) [inline doc/method/sources.md]: this reminds me I should list somewhere sources requested by curators but not yet added: paleobio DB, bird list, plant list, fishbase.  also must mention that we're prohibited from using IUCN, the only recent mammal list (maybe I already say that)

**Comment by kcranston** [comment]
I've made some changes, including the start of an abstract. It is starting to come together! I think you are on the right track for results - take a look at what I added in [this commit](https://github.com/OpenTreeOfLife/reference-taxonomy/pull/246/commits/610d92e5f796996e577907fe5e7a4d5188582e63)

**Pull request merged**